test/Tests/src/Semantic/Warnings_Spec.enso

```diff
@@ -157,4 +157,13 @@ spec = Test.group "Dataflow Warnings" <|
         result = x.is_static_nothing x
         result . should_equal False
 
+    Test.specify "should be allowed in Vector" <|
```

Comment:
Unlikely this will work. This is a subject of https://www.pivotaltracker.com/n/projects/2539304/stories/182194556. The fix for the latter might actually render this change obsolete.

Reply:
Ok, I hoped that maybe you managed to fix two birds with one ~~stone~~ _bugfix_.

But then, the latter may wait for its turn :)

Reply:
It is now